digid_eherkenning/models/base.py

```diff
@@ -203,6 +224,28 @@ def __init__(self, *args, **kwargs):
     def __str__(self):
         return force_str(self._meta.verbose_name)
 
+    @property
+    def technical_contact_person(self) -> ContactPerson | None:
+        if (telephone := self.technical_contact_person_telephone) and (
+            email := self.technical_contact_person_email
+        ):
+            return {
+                "telephoneNumber": telephone,
+                "emailAddress": email,
+            }
+        return None
+
+    @property
+    def administrative_contact_person(self) -> ContactPerson | None:
+        if (telephone := self.administrative_contact_person_telephone) and (
+            email := self.administrative_contact_person_email
+        ):
+            return {
+                "telephoneNumber": telephone,
+                "emailAddress": email,
+            }
+        return None
```

Comment:
I see that in the docs it says (https://afsprakenstelsel.etoegang.nl/Startpagina/AS1.24b/metadata-for-participants):

_The EntityDescriptor MUST contain **one or more** elements of the type ContactPerson containing an administrative non-personal name, email address, and telephone number that can be contacted by participants or the Beheerorganisatie in the event of an incident._

Do we need some validation that at least one of the administrative/technical contactPerson is filled in?

Reply:
I think we need to leave it as is, because the requirements are less strict for the test network and that would make it harder to getting started. Ultimately the brokers are responsible for validating the metadata so they can and should point out that this information was not included.